Makefile

```diff
@@ -54,4 +55,4 @@ frontend:
 
 .PHONY: integration
 integration: integration.test bldr
-	cd internal/pkg/integration && PATH="$$PWD/../../../out:$$PATH"  integration.test -test.v
+	cd internal/pkg/integration && PATH="$$PWD/../../../out:$$PATH"  integration.test -test.v -test.run $(RUN_TESTS)
```

Comment:
this allows to run subset of the tests, e.g.:

```
make integration RUN_TESTS=TestIntegration/variables/docker
```

Test name includes both test name and runner type, so we can limit the scope in a pretty efficient way.